packages/mobile/src/app/verificationMigration.test.ts

```diff
@@ -13,142 +14,133 @@ import { mockAccount, mockE164Number, mockE164NumberPepper } from 'test/values'
 const now = Date.now()
 Date.now = jest.fn(() => now)
 
+const mockMTW = mockAccount
+
 describe('verificationMigration', () => {
   beforeEach(async () => {
     jest.clearAllMocks()
   })
 
-  it("doesn't change anything if mtwAddress is present and verified", async () => {
+  async function commonProvides({
```

Comment:
I didn't actually change the tests (or didn't intend to) they were just hard to read because of all the repeated code so I created this function so that each test only passes the relevant fields